### DISCUSSION TITLE
SNMP service

### DISCUSSION THREAD

**Discussion by dubefab** [akvorado/akvorado#512] [Q&A] [answered]
I have installed Akvorado on the same VM that runs LibreNMS, could it be interfering with it?
Asking because I am not getting traffic into Akvorado.
Thanks

**Comment by vincentbernat** [comment, marked as answer]
Akvorado is running an SNMP manager (the client part), so it can run from the same server as LibreNMS. Look at the troubleshooting documentation to find what the problem could be.

**Reply by dubefab** [reply]
Thanks for the prompt answer.
All I did basically is install akvorado with the docker-composer.
The ip:8081 webpage loads but all the commands I do from the troubleshooting page return nothing.
`curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet'`

But running:
`curl -s http://127.0.0.1:8080/api/v0/inlet/flows\?limit=1`
returns:
404 page not found

**Reply by dubefab** [reply]
I have tried configuring a Cisco firewall to send netflow traffic.

**Reply by vincentbernat** [reply]
Check the output of `docker-compose ps`. Likely you have the inlet component or the orchestrator-component restarting. Check `docker-compose logs akvorado-inlet` for reasons.

**Reply by absmith** [reply]
I am having a similar issue. ps shows all services running no restarts. 
I feel it never tries to do the snmp polling. 
docker-compose logs akvorado-inlet
doesn't show much other than hits on the healthcheck and metrics urls. I do see starting SNMP component message
